tests/conftest.py

```diff
@@ -240,6 +240,20 @@ def application_docker_image_configs() -> Mapping[str, Dict[str, Any]]:
         "nodejs": {
             "": dict(buildargs={"NODE_RUNTIME_FLAGS": "--perf-prof --interpreted-frames-native-stack"}),
             "without-flags": dict(buildargs={"NODE_RUNTIME_FLAGS": ""}),
+            "10-alpine": dict(dockerfile="matrix.Dockerfile", buildargs={"NODE_IMAGE_TAG": "10.24.1-alpine"}),
```

Comment:
Instead of `10-alpine` and `10-slim`, better names are `10-musl` and `10-glibc` - that's what we're testing after all, and `slim` / `alpine` images are only the implemntation detail. Let's change for all